lightning/src/routing/gossip.rs

```diff
@@ -752,7 +752,7 @@ impl ChannelInfo {
 
 	/// Returns a [`DirectedChannelInfo`] for the channel directed from the given `source` to a
 	/// returned `target`, or `None` if `source` is not one of the channel's counterparties.
-	pub fn as_directed_from(&self, source: &NodeId) -> Option<(DirectedChannelInfo, &NodeId)> {
+	pub(crate) fn as_directed_from(&self, source: &NodeId) -> Option<(DirectedChannelInfo, &NodeId)> {
```

Comment:
do we need a separate commit for this? I think it can go with as_directed_to. Especially considering there's a bunch of unrelated stuff in between the two commits.